Jakefile

```diff
@@ -31,6 +31,19 @@ task('minify', function () {
   console.log('Minification completed.');
 });
 
+task('doc', function () {
+  exec('./node_modules/.bin/rimraf out');
```

Comment:
Since Windows cmd.exe doesn't have an equivalent for shebang line, npm is smart enough to create `.cmd` files in node_modules/.bin that invoke node.